collect_app/src/main/java/org/odk/collect/android/projects/AddProjectDialog.kt

```diff
@@ -60,7 +60,7 @@ class AddProjectDialog : MaterialFullScreenDialogFragment() {
         }
 
         binding.addButton.setOnClickListener {
-            projectsRepository.add(getProjectName(), getProjectIcon())
+            projectsRepository.add(getProjectName(), getProjectIcon(), getProjectColor())
```

Comment:
It feels like it would be good at this point to have `ProjectsRepository#add` just take a `Project` as an argument instead of the individual components. What do you think?

Reply:
I agree, fixed.